### PR TITLE
Map Explorer (Episode 4) - Improved GeoData Layers

### PR DESCRIPTION
**Pull request by shaunanoordin**
Features
- Gorongosa National Park geodata layer restyled so it's only a border.
- Vegetation geodata layer restyled to support different colours for each terrain type.
  - No new geodata layers added to avoid noisiness on Map Explorer... for now.
- Legend for Vegetation geodata layer added.
- The "Layers" submenu, on the upper right, is now permanently expanded to make it SUPER obvious to users that they can toggle layers.

screen shot 2016-08-02 at 15 30 56
Screen layout on a MacBook Pro (Retina, 13-inch, Early 2015). Also note use of colours; the mountain is pink-ish because I simply ran out of contrasting colours that work at such a low opacity.

Known Issue
- CartoDB Data layer appears below the GeoJSON overlay layers.
- This is because the CartoDB is technically a raster 'tile' layer that is grouped in Leaflet's `tilePane` (together with the base map layer), instead of the 'higher' (z-index-wise) `overlayPane` where the GeoJSON layers appear. So... rearranging them would be impossible unless I move the CartoDB Data layer out  of the `tilePane` group somehow.
- Workaround: GeoJSON overlay layers use visual tricks to make it less obvious that the CartoDB data points are beneath them. (Low opacity, etc)

This PR is ready for review. Improves on closed issue #247 and the 'add layers item' on milestone list #242

### CODE REVIEW

File: src/modules/common/containers/MapVisuals.jsx [hunk 1]
```diff
@@ -128,12 +128,25 @@ class MapVisuals extends Component {
     const legend = L.control({position: 'bottomright'});
     legend.onAdd = (map) => {
       const div = L.DomUtil.create('div', 'info legend');
-      div.innerHTML +=
+      div.innerHTML =
```
Comment (simoneduca) [inline src/modules/common/containers/MapVisuals.jsx]: All this html should be put into a presentational component instead.

File: src/modules/common/containers/MapVisuals.jsx [hunk 2]
```diff
@@ -128,12 +128,25 @@ class MapVisuals extends Component {
...
         '<div><svg height="10" width="10"><circle cx="5" cy="5" r="5" fill="#666" /></svg> : Camera with no images</div>' +
         '<div><svg height="10" width="10"><circle cx="5" cy="5" r="5" fill="#f93" /></svg> : Camera with images (click to view)</div>';
       return div;
     };
     legend.addTo(this.state.map);
+    
+    //Bonus: Add (vegetation) legends to map
+    const vegetationLegend = L.control({position: 'bottomright'});
+    vegetationLegend.onAdd = (map) => {
+      const div = L.DomUtil.create('div', 'info legend');
```
Comment (simoneduca) [inline src/modules/common/containers/MapVisuals.jsx]: same as above

**Comment by shaunanoordin** [comment]
@simoneduca I've addressed the feedback on the Map Legends

**Comment by simoneduca** [comment]
LGTM 👍

**Pull request merged**